.gitignore

```diff
@@ -30,6 +30,7 @@ tester2.rb
 tester.html.erb
 tester2.html.erb
 tester.js.erb
+misc.sql
```

Comment:
What is this?

Reply:
just a scratch pad. we should just move this stuff to a tmp folder and exclude that